client/src/components/body/lab/LabGeneration.js

```diff
@@ -81,26 +82,58 @@ const LabGeneration = (props) => {
         );
       } else {
         return (
-          <p className="module__no_labs">You have no labs for this section.</p>
+          <>
+            {!search && (
+              <p className="module__no_labs">
+                You have no labs for this section.
```

Comment:
What does "this section" mean? I would say something like "You have no in-progress or completed labs".

Reply:
![Screenshot 2024-10-25 at 1 03 56 PM](https://github.com/user-attachments/assets/04660da2-4943-448d-8405-56aaf6d9ba7f)
Looking at the figma, this is what the text should say